lib/ast-link-to-transform.js

```diff
@@ -91,20 +74,22 @@ class AngleBracketLinkToPolyfill {
           let props = attributes
             .filter(({ name }) => name.charAt(0) === '@' && !reservedProps.includes(name))
             .map(attribute => Object.assign({}, attribute, { name: attribute.name.slice(1) }));
-          let attrs = attributes
-            .filter(({ name }) => supportsAttribute(name, supportedHTMLAttributes))
+          let mappedAttrs = attributes
```

Comment:
These were separated from the other attributes because grouping them into `ANGLE_ATTRS` would cause them to get processed incorrectly by this code:

https://github.com/ember-polyfills/ember-angle-bracket-invocation-polyfill/blob/master/vendor/angle-bracket-invocation-polyfill/runtime-polyfill.js#L230

If there is a better way to do this, I'm open to suggestion, but I essentially wanted to ensure that `ariaRole` was passed as a property and not an attribute.